test/integration/helpers.sh

```diff
@@ -153,6 +153,15 @@ tpm2_abrmd_pid=""
 tpm2_tcti_opts=""
 tpm2tools_tcti=""
 
+sock_tool="unknow"
+OS=$(uname)
+
+if [ "$OS" == "Linux" ]; then
+    sock_tool="netstat -lntp"
+elif [ "$OS" == "FreeBSD" ]; then
+    sock_tool="sockstat -l"
```

Comment:
FreeBSD might have the same problem with the simulator occasionally binding only to IPv6, so using `sockstat -l4` would be more robust here as well (and is supported according to the [FreeBSD man page](https://www.freebsd.org/cgi/man.cgi?query=sockstat&manpath=FreeBSD+12.0-RELEASE+and+Ports)).